tests/test_task.py

```diff
@@ -267,19 +266,32 @@ def test_build_trials_seq_shuffling():
     _ = task_shuffled.generate_trials(ntrials=NTRIALS)
     _ = task_not_shuffled.generate_trials(ntrials=NTRIALS)
 
-    # Verify that the generated sequences are shuffled or not shuffled accordingly
     assert task_shuffled._modality_seq.shape == task_not_shuffled._modality_seq.shape
-    assert not np.array_equal(task_shuffled._modality_seq, task_not_shuffled._modality_seq)
+
+    keys = list(task_shuffled._session.keys())
+    first_occurrence = list(task_not_shuffled._modality_seq).index(keys[1])
+    assert (
```

Comment:
In which cases is this going to be False? I didn't get the point of this assert

Reply:
It's false if there's a "v" after an "a". I added a comment to the code to make this clearer.